convertTimeToBinaryTimeString.js

```diff
@@ -0,0 +1,5 @@
+const convertTimeToBinaryTimeString = (time) => {
+  return time
+    .split('')
+    .map((time) => parseInt(time, 10).toString(2).padStart(4, '0'));
+};
```

Comment:
```suggestion
const convertTimeToBinaryTimeString = (times) => {
  return times
    .split('')
    .map((time) => parseInt(time, 10).toString(2).padStart(4, '0'));
};
```

It better to not use the same name in same function.